src/components/ReportActionItem/ReportActionItemImage.tsx

```diff
@@ -82,6 +83,7 @@ function ReportActionItemImage({
             fallbackIcon: Expensicons.Receipt,
             fallbackIconSize: isSingleImage ? variables.iconSizeSuperLarge : variables.iconSizeExtraLarge,
             isAuthTokenRequired: true,
+            shouldUseInitialObjectPosition: isDistanceRequest ?? false,
```

Comment:
Why is `?? false` needed here? Won't `isDistanceRequest` always be true or false?

Reply:
Gotta love JS :D I expected the `&&` to do some kind of boolean casting for us, seems it doesn't haha

<img width="233" alt="Screenshot 2024-05-06 at 10 11 28 AM" src="https://github.com/Expensify/App/assets/3885503/f10b37bf-bf71-4e63-bd28-297cb5dff24a">